how-to-docs/namespace-how-to.md

```diff
@@ -1,10 +1,10 @@
 ## How do I create a namespace in the UK cluster?
 Please visit the [ACP-Kube](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube) repo on Gitlab to create a namespace.
 
-+ Clusters are defined by the folders within the `namespaces/` directory. This directory contains all defined namespaces for each given cluster.
++ Clusters are defined by the folders within the `policies/` directory, with each cluster having a subdirectory labelled `namespaces`, this directory contains all defined namespaces for each given cluster.
 
 + Namespaces should be defined in the following format within this repository:
-`namespaces/<cluster-name>/<namespace.yaml>`
+`policies/<cluster-name>/namespaces/<namespace.yaml>`
```

Comment:
i think the directory path is wrong, should be `policies/clusters/<cluster-name>/namespaces/<namespace.yaml>`
e.g. `acp-kube/policies/clusters/acp-ops/namespaces/drone.yaml`